### PR TITLE
Add Cloud Engingeering Summit workshops

### PR DESCRIPTION
**Pull request by zchase**
This pR adds the workshops for the Cloud Engineering Summit and lists them on the landing page as well.

### CODE REVIEW

File: themes/default/content/resources/getting-started-with-aws/index.md [hunk 1]
```diff
@@ -77,8 +77,8 @@ main:
     presenters:
         - name: "Kat Cosgrove"
           role: "Staff Developer Advocate, Pulumi"
-        - name: "Sophia Parafina"
-          role: "Technical Marketing Manager"
+        - name: "Matty Stratton"
+          role: "Staff Developer Advocate, Pulumi"
```
Comment (nimbinatus) [inline themes/default/content/resources/getting-started-with-aws/index.md:81]: I think I'm the one presenting this time for AWS for Cloud Engineering Summit, and I think Marina from AWS is joining me. Is this one form for all of the upcoming AWS workshops? Should I just add us both here?

File: themes/default/content/cloud-engineering-summit/_index.md [hunk 2]
```diff
@@ -8,6 +8,49 @@ layout: cloud-engineering-summit
 hubspot_form_id: 15b92d63-158e-4fc0-8b2c-c4bf3fbfee27
 sessionize_id: wbl7wd7l
 
+workshops:
```
Comment (nimbinatus) [inline themes/default/content/cloud-engineering-summit/_index.md:11]: Missing the NGINX one (it's in Sessionize but not scheduled yet.):
> Going from Zero to Production in Kubernetes with NGINX and Pulumi
>
>In this workshop, we will highlight the NGINX MARA project and show you how you can stand up an entire application infrastructure completely automated with Pulumi.  We will walk through all of the different components and tooling to build a Kubernetes based microservices application deployment, with all of the required tools in place leverage production ready best practices. 

Speakers:
* Damian Curry
* Elijah Zupancic

Workshop is on 10/21; not sure when exactly.

**Review by nimbinatus — CHANGES_REQUESTED** [review]

File: themes/default/content/cloud-engineering-summit/_index.md (hunk 2 above)
Comment (mattstratton) [reply themes/default/content/cloud-engineering-summit/_index.md:11]: It's actually been scheduled in sessionize and is on the agenda I believe

File: themes/default/content/resources/getting-started-with-aws/index.md (hunk 1 above)
Comment (zchase) [reply themes/default/content/resources/getting-started-with-aws/index.md:81]: 👍

File: themes/default/content/cloud-engineering-summit/_index.md (hunk 2 above)
Comment (zchase) [reply themes/default/content/cloud-engineering-summit/_index.md:11]: 👍

**Review by katcosgrove — APPROVED** [review]

File: themes/default/content/resources/getting-started-with-aws/index.md (hunk 1 above)
Comment (katcosgrove) [reply themes/default/content/resources/getting-started-with-aws/index.md]: @zchase I think the name on this should be Laura Santamaria

**Review by nimbinatus — APPROVED** [review]

**Pull request merged**